jenkins/jobs/openstack_ci_image_building.pipeline

```diff
@@ -0,0 +1,80 @@
+ci_git_credential_id = "metal3-jenkins-github-token"
+ci_git_branch = "main"
+ci_git_url = "https://github.com/metal3-io/project-infra.git"
+
+pipeline {
+  agent { label 'metal3ci-jenkins' }
+  options { ansiColor('xterm') }
+  environment {
+    IMAGE_OS = "${IMAGE_OS}"
+    METAL3_CI_USER="metal3ci"
+    OS_AUTH_URL="https://kna1.citycloud.com:5000"
+    OS_USER_DOMAIN_NAME="CCP_Domain_37137"
+    OS_PROJECT_DOMAIN_NAME="CCP_Domain_37137"
+    OS_REGION_NAME="Kna1"
```

Comment:
For Fra1 would we be building images also or would be build once and push twice in both regions? In any case should this var be dictated by jjb rather ?

Reply:
Fair enough. lgtm
/hold cancel